Gemfile

```diff
@@ -117,6 +117,8 @@ gem 'flipper'
 gem 'flipper-active_record'
 gem 'flipper-ui'
 
+gem "activesupport"
```

Comment:
`ActiveSupport` is already included in Rails, we don't need to add it to the Gemfile.

Reply:
Ok! Removed.